src/components/Landing/index.tsx

```diff
@@ -211,13 +265,18 @@ export default function Landing() {
               {smallHeader(t`Projects using Juicebox`)}
               <div style={{ marginTop: 20 }}>
                 {previewProjects ? (
-                  <ProjectsGrid projects={previewProjects} list />
+                  <Grid
+                    children={previewProjects.map(p => (
+                      <ProjectCard project={p} />
+                    ))}
+                    list
+                  />
```

Comment:
If you're going to use `children`, you should use it syntactically correct, ie:

```suggestion
                  <Grid list>
                    {previewProjects.map(p => (
                      <ProjectCard key={p.uri} project={p} />
                    ))}
                  </Grid>
```

Reply:
For sure 👍